cmd/dtmate/cmd/root.go

```diff
@@ -37,9 +49,25 @@ var rootCmd = &cobra.Command{
 	Use:     "dtmate",
 	Short:   "dtmate: output the difference between date, time or duration",
 	Version: DateTimeMate.ModVersion,
+	Run: func(cmd *cobra.Command, args []string) {
+		if optRootShowExamples {
+			ShowExamples()
+			os.Exit(0)
+		}
+
+		// if no arguments are provided and no flags are set, show help
+		if len(args) == 0 && !cmd.Flags().Changed("examples") {
+			cmd.Help()
+		}
+	},
 }
 
 var optRootNoNewline bool
+var optRootShowExamples bool
+var readmeExamplesRegex = regexp.MustCompile(`(?ms)## Command Line Examples.*?shell\n(.*?)` + "```")
+
+//go:embed README.md
+var readmeFS embed.FS
```

Comment:
I would have used


```suggestion
var readmeContent string
```

It would have been simpler

Reply:
I was just following some examples I studied since I've never used file embedding before.